modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1221,7 +1221,7 @@ Included when xref:configuration/configuration-settings.adoc#config_db.logs.quer
 | Connection details.
 
 | database
-| The database name on which the query is run.
+| The database name on which the query is run. This field will be `<none>` if it is not possible to parse and route the query to the correct database.
```

Comment:
```suggestion
| The database name on which the query is run. This field will be `<none>` if it is impossible to parse and route the query to the correct database.
```

Reply:
For some reason it felt weird to say `impossibe` (don't ask me why 😄). Would it be okay with the change I made to:

"This field will be `<none>` if the query cannot be parsed and routed to a database."